src/it/overrideQubeMinVersion/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.codehaus.sonar</groupId>
+  <artifactId>test</artifactId>
+  <version>1.0</version>
+  <packaging>sonar-plugin</packaging>
+  <name>Package dependencies</name>
+  <description>Package dependencies</description>
+
+  <properties>
+    <sonar.qubeMinVersion>5.1</sonar.qubeMinVersion>
```

Comment:
I feel the naming should be more explicit. We could maybe use a specific prefix for packaging plugin 'sonar.packaging'. Something like sonar.packaging.minSupportedSonarQubeVersion. Then we could use sonar.packaging.minSupportedSonarLintVersion for SL